test/booklist.spec.js

```diff
@@ -3,6 +3,11 @@ const expect = require("chai").expect;
 const getUrlByParameter = require("../scripts/booklist").getUrlByParameter;
 
 describe('booklist', () =>{
+  it('should return type string', () =>{
+    dom.reconfigure({url: "chrome-extension://hkahpanhaccgppbidkekeijffcdppdan/booklist.html?url=https://en.wikipedia.org/wiki/Easter_Island"});
```

Comment:
You test the same function two times with the same argument!! What's the point?

Reply:
I mean, if its equal to a string, its going `.to.be.a("string")`, there's no meaning in doing this test.

Reply:
I was just following some convention I saw on someone's blog post, however, if you feel it is redundant, I'm happy to remove it.